.github/CODEOWNERS

```diff
@@ -1,8 +1,11 @@
+# DO NOT CHANGE. This file is being managed from a central repository
+# To know more simply visit https://github.com/honestbank/.github/blob/main/docs/about.md
+
 # This is a comment.
 # Each line is a file pattern followed by one or more owners.
 
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-*       @honestbank/devops-engineers
+*       @honestbank/devops-engineers @honestbank/backend-engineers
```

Comment:
Unknown owner on line 11: make sure the team @honestbank/backend-engineers exists and has write access to the repository

… @honestbank/devops-engineers @honestbank/backend-engineers

Reply:
@MXfive done